lib/signaling/v2/peerconnection.js

```diff
@@ -387,9 +396,13 @@ class PeerConnectionV2 extends StateMachine {
       this._isRestartingIce = false;
       this._shouldRestartIce = true;
     }
-    return Promise.resolve().then(() => this._setLocalDescription({ type: 'rollback' }))
-      .then(() => this._answer(offer))
-      .then(didReoffer => didReoffer ? Promise.resolve() : this._offer());
+    return Promise.resolve().then(() => {
+      return this._setLocalDescription({ type: 'rollback' });
+    }).then(() => {
+      return this._answer(offer);
```

Comment:
This line answers above question. But could we set `needsAnswer = false` here right after rollback? and remove it from `_answer(...)` function?

Reply:
@syerrapragada `_answer()` may re-offer, so I think the existing location is the ideal place for setting `_needAnswer` to false.

Reply:
We have already rolled back the previous offer successfully so do we still need to wait for an answer?